tests/e2e/util.go

```diff
@@ -2360,17 +2404,55 @@ func getPvFromSupervisorCluster(pvcName string) *v1.PersistentVolume {
 	return svcPV
 }
 
-func verifyFilesExistOnVSphereVolume(namespace string, podName string, filePaths ...string) {
+// verfiy File exists or not in vSphere Volume
+func waitAndVerifyFilesExistOnVSphereVolume(namespace string, podName string, poll, timeout time.Duration, filePaths ...string) {
+	var err error
 	for _, filePath := range filePaths {
-		_, err := framework.RunKubectl(namespace, "exec", fmt.Sprintf("--namespace=%s", namespace),
-			podName, "--", "/bin/ls", filePath)
+		if windowsEnv {
+			for start := time.Now(); time.Since(start) < timeout; time.Sleep(poll) {
+				_, err = framework.LookForStringInPodExec(namespace, podName,
+					[]string{"powershell.exe", "Test-Path -path", filePath}, "", time.Minute)
+				if err == nil {
+					break
+				} else if err != nil {
+					framework.Logf("File %s doesn't exist", filePath)
+					continue
+				}
+			}
+		} else {
+			_, err = framework.RunKubectl(namespace, "exec", fmt.Sprintf("--namespace=%s", namespace),
+				podName, "--", "/bin/ls", filePath)
+		}
 		framework.ExpectNoError(err, fmt.Sprintf("failed to verify file: %q on the pod: %q", filePath, podName))
 	}
 }
 
+// verify File System type on vSphere volume
+func verifyFsTypeOnVsphereVolume(namespace string, podName string, expectedContent string, filePaths ...string) {
+	var err error
+	for _, filePath := range filePaths {
+		if windowsEnv {
+			filePath = filePath + ".txt"
+			waitAndVerifyFilesExistOnVSphereVolume(namespace, podName, poll, pollTimeoutShort, filePath)
```

Comment:
add a TODO note here to remove this after we add logic for disabling disk caching in windows